### PR TITLE
Fixed #34199 -- Added example to StringAgg docs.

### PR DESCRIPTION
**Pull request by sdolemelipone**
As there wasn't any. Also thought it would be useful to have a ManyToMany example.

### CODE REVIEW

File: docs/ref/contrib/postgres/aggregates.txt [hunk 1]
```diff
@@ -208,6 +208,31 @@ General-purpose aggregation functions
 
         Examples are the same as for :attr:`ArrayAgg.ordering`.
 
+    Usage example::
+
+        class Publication(models.Model):
+            title = models.CharField(max_length=30)
+
+        class Article(model.Model):
```
Comment (shangxiao) [inline docs/ref/contrib/postgres/aggregates.txt:216]:
```suggestion
        class Article(models.Model):
```

**Review by shangxiao — CHANGES_REQUESTED** [review]
Thanks for that looks useful ❤️

There's a typo that needs fixing though before handing over to felixx.

**Comment by shangxiao** [comment]
Nitpick: also may want to clarify in the commit message that it's adding an example to the docs, ie not adding documentation 🤔

File: docs/ref/contrib/postgres/aggregates.txt (hunk 1 above)
Comment (sdolemelipone) [reply docs/ref/contrib/postgres/aggregates.txt]: Thanks! That's what I get from copy pasting from the JSONBAgg function, I have corrected that typo as well.

**Comment by sdolemelipone** [comment]
> Nitpick: also may want to clarify in the commit message that it's adding an example to the docs, ie not adding documentation thinking

Corrected!

**Comment by sdolemelipone** [comment]
Just fixed another minor typo....sorry, I'm done making changes now! :-)

**Comment by felixxm** [comment]
@sdolemelipone Thanks :+1:

**Pull request merged**